client/src/partials/patients/registry/registry.js

```diff
@@ -49,6 +51,29 @@ function PatientRegistryController($translate, Patients) {
     loadGrid();
   }
 
+  // Search and filter data in Patiens Registry
+  function search() {
```

Comment:
Could you put the modal in the `PatientService`?  That way, it can be called from multiple pages in necessary.  Something like:
```js
// Patients is the PatientService
function search() {
    Patients.openSearchModal()
    .then(function (data) {
      // etc..
    });
```